packages/snaps-controllers/src/snaps/SnapController.ts

```diff
@@ -2231,6 +2231,10 @@ export class SnapController extends BaseController<
         this.#calculatePermissionsChange(snapId, processedPermissions);
 
       this.#updateApproval(pendingApproval.id, {
```

Comment:
Please add a test or an assertion to an existing test testing that `initialConnections` is part of the `requestState`.

Reply:
Added.